src/index.tsx

```diff
@@ -50,39 +51,64 @@ const containerStyle: React.CSSProperties = {
   zIndex: 100001,
 };
 
-const Modal: React.FC<ModalProps> = ({ children, isOpen = false, close, elementId = 'root' }) => {
+const Modal: React.FC<ModalProps> = ({ children, isOpen = false, close, elementId = 'root', options = {} }) => {
+  const { preventScroll = false } = options;
+  const modalRef = useRef<HTMLDivElement>(null);
+
+  const onOpen = useCallback(() => {
+    if (preventScroll) {
+      if (modalRef.current != null) {
```

Comment:
`!==` is better.

Reply:
@shibe97 
Thank you for reviewing.
I used it to mean `!== null` and `!== undefined`.
The condition of `undefined` cannot occur, so it has been fixed.